src/Pomelo.EntityFrameworkCore.MySql/Extensions/MySqlDbContextOptionsExtensions.cs

```diff
@@ -87,5 +90,25 @@ private static MySqlOptionsExtension GetOrCreateExtension(DbContextOptionsBuilde
                 ? new MySqlOptionsExtension(existing)
                 : new MySqlOptionsExtension();
         }
+
+        private static bool IsAbleToUseDateTime6(string connectionStr)
```

Comment:
The microsoft folks gave us a suggestion: https://github.com/aspnet/EntityFramework/issues/7275